doc/source/whatsnew/v1.1.0.rst

```diff
@@ -342,6 +342,21 @@ Other enhancements
 - ``compute.use_numba`` now exists as a configuration option that utilizes the numba engine when available (:issue:`33966`, :issue:`35374`)
 - :meth:`Series.plot` now supports asymmetric error bars. Previously, if :meth:`Series.plot` received a "2xN" array with error values for ``yerr`` and/or ``xerr``, the left/lower values (first row) were mirrored, while the right/upper values (second row) were ignored. Now, the first row represents the left/lower error values and the second row the right/upper error values. (:issue:`9536`)
 
+
+.. ---------------------------------------------------------------------------
+
+.. _whatsnew_110.api_breaking:
+
+Backwards incompatible API changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_110.api_breaking._testing.assert_frame_equal:
```

Comment:
yeah i am not sure it is useful to put in docs in the past, these won't be rebuilt for older versions.
best option is simply to put it in 1.2.1 docs and in the note itself say this starts affecting in 1.1.x

Reply:
Will move to 1.2.1.

Reply:
What section would you like this under?

Reply:
> yeah i am not sure it is useful to put in docs in the past, these won't be rebuilt for older versions.

indeed, we don't rebuild the archived docs for the previous versions, but we do rebuild the release notes for previous versions and these will be the release notes on stable.

> best option is simply to put it in 1.2.1 docs and in the note itself say this starts affecting in 1.1.x

difficult call since someone with issues with an older version, may go to the stable docs to look through the release history. (I know I do, rather than going the the docs archive of the actual version)

I was initially confused by the release note in 1.2.1, astonished actually that there was a breaking change. It was not clear to me that the breaking change occurred in 1.1.0

Reply:
@simonjayhawkins @jreback would you like a follow up PR to move this back to 1.1.0.rst? I'd suggest merging #38893 first to simplicity sake.